src/book-byen/__tests__/book-byen.stories.tsx

```diff
@@ -36,6 +36,7 @@ const Template: ComponentStory<typeof BookByen> = ({ ...args }: any) => {
     },
   };
 
+  // eslint-disable-next-line react/jsx-props-no-spreading
   return <BookByen {...props} />;
```

Comment:
You could also just pick the three props that BookByen [uses](https://github.com/kkos2/display-templates/blob/kk-develop/src/book-byen/book-byen.tsx#L41) - or am I missing something?